src/solver/gemm_bwd.cpp

```diff
@@ -43,6 +43,7 @@
 MIOPEN_DECLARE_ENV_VAR(MIOPEN_CONV_PRECISE_ROCBLAS_TIMING)
 
 #define WORKAROUND_MIOPENGEMM_ISSUE_59 1
+#define WORKAROUND_MIOPENGEMM_ISSUE_2474 1
```

Comment:
```suggestion
#define WORKAROUND_ISSUE_2474 1
```
We do not use MIOpenGemm (GEMM backend for OpenCL) anymore.

Reply:
done in https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2477/commits/fa5a598adc02d6a381d50ceb80fd41d7cf68c9b7

Reply:
[resolved]